AdminCore.Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using System.Linq;
+using AdminCore.Constants;
+
+namespace AdminCore.Extensions
+{
+    public static class StringExtensions
+    {
+        public static List<string> GetContentFromPolicy(this string policyName)
```

Comment:
is it possible to make context of this function more generic, so the extension can be comfortably reused for similar purposes? why is this functionality not tested?

Reply:
Made more generic and added tests.